File: java/air-io-network/src/main/java/org/feuyeux/air/io/network/udpctrl/controlinfo/KeyControlInfo.java
package org.feuyeux.air.io.network.udpctrl.controlinfo;

/**
 * Created by erichan on 2/10/14.
 */
public class KeyControlInfo implements ControlInfo {
    private String keyPress;

    public KeyControlInfo() {
    }

    public KeyControlInfo(String keyPress) {
        this.keyPress = keyPress;
    }

    public String getKeyPress() {
        return keyPress;
    }

    public void setKeyPress(String keyPress) {
        this.keyPress = keyPress;
    }

    @Override
    public String toString() {
        return getKeyPress();
    }
}

File: ios/learning/UdpCmd/UdpCmd/KeyControlInfo.m
//
//  KeyControlInfo.m
//  UdpCmd
//
//  Created by feuyeux@gmail.com on 14-2-13.
//  Copyright (c) 2014年 feuyeux@gmail.com. All rights reserved.
//

#import "KeyControlInfo.h"

@implementation KeyControlInfo

-(NSString*) toString{
    return _keyPress;
}

-(KeyControlInfo *)initWithKeyPress:(NSString*)keyPress
{
    self=[super init];
    [self setKeyPress:keyPress];
    return self;
}
@end
